extensions/wikia/Follow/FollowEmailTask.class.php

```diff
@@ -4,9 +4,18 @@
 
 class FollowEmailTask extends BaseTask {
 
-	public function emailFollowNotifications( $aWatchers, $iUserId, $iNamespace, $sMessage, $sAction ) {
-		$now = wfTimestampNow();
-		$oUser = User::newFromId( $iUserId );
+	public function emailFollowNotifications( $initiatingUser, $aWatchers, $iUserId, $iNamespace, $sMessage, $sAction ) {
+		$wg = F::app()->wg;
+		$wg->DBname = WikiFactory::IDtoDB( $this->getWikiId() );
+		$wg->Server = trim( WikiFactory::DBtoUrl( F::app()->wg->DBname ), '/' );
+
+		if ( !empty( $wg->DevelEnvironment ) ) {
+			$wg->Server = WikiFactory::getLocalEnvURL( $wg->Server );
+		}
+
+		$wg->User = User::newFromId( $initiatingUser );
```

Comment:
Both ```$wgUser``` and ```$wgServer``` were needed to construct the right links.  If adding the ```wikiId``` when creating the task was supposed to set ```$wgServer```, it wasn't working here...